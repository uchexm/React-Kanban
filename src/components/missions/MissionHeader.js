import React from 'react';

import './missionHeader.css';

export default function MissionHeader() {
  return (
    <table>
      <thead>
        <tr>
          <th>Mission</th>
          <th>Description</th>
          <th style={{ paddingLeft: '30px' }}>Status</th>
          <th>  </th>
        </tr>
      </thead>
    </table>
  );
}
