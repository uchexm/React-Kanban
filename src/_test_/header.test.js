import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import MissionHeader from '../components/missions/MissionHeader';

it('Test if footer rendered', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <MissionHeader />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
