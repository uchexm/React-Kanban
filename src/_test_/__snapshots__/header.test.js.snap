// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if footer rendered 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Mission
      </th>
      <th>
        Description
      </th>
      <th
        style={
          Object {
            "paddingLeft": "30px",
          }
        }
      >
        Status
      </th>
      <th>
          
      </th>
    </tr>
  </thead>
</table>
`;
